lib/browser/tag/tag.js

```diff
@@ -68,7 +68,10 @@ function Tag(impl, conf, innerHTML) {
       mix = 'string' == typeof mix ? riot.mixin(mix) : mix
       each(Object.keys(mix), function(key) {
         // bind methods to self
-        self[key] = 'function' == typeof mix[key] ? mix[key].bind(self) : mix[key]
+        self[key] = 'function' == typeof mix[key] && 'init' != key
```

Comment:
@rsbondi Initializer `init` is skipped and run just after the mixin is loaded. I think it will avoid the multi-mixin problem. How about?